### ISSUE TITLE
[Feature]: replace Halloween logo with original 

### ISSUE THREAD

**Issue by itsalaidbacklife** [cuttle-cards/cuttle#199]
### Feature Summary

We replaced the logo with a Halloween version during the month of October. We should replace it with the original now that Halloween is over

### Detailed Description

This is a simple revert of #166. Can be opened before Halloween to count for Hacktoberfest and will be merged after Nov 1.

**Comment by surajrane42** [comment]
@itsalaidbacklife Can you assign this to me? I would like to work on it

**Comment by itsalaidbacklife** [comment]
Of course, thank you! Let me know if you have any questions or difficulties

**Comment by surajrane42** [comment]
@itsalaidbacklife Thank you. I have fixed & create a PR. Please check
